source/conf.py

```diff
@@ -44,6 +44,7 @@
 ps57recommended = '5.7.35'
 ps80recommended = '8.0.25'
```

Comment:
ps80recommended='8.0.28'
and for ps57 we don't support right now and I don't think we will.